torch/csrc/utils/python_arg_parser.cpp

```diff
@@ -348,13 +349,28 @@ bool is_tensor_and_append_overloaded(PyObject* obj, std::vector<py::handle>* ove
   return false;
 }
 
-bool is_tensor_list_and_append_overloaded(PyObject* obj, std::vector<py::handle>* overloaded_args, int argnum, bool throw_error) {
+bool is_scalar_list_and_append_overloaded(PyObject* obj, int argnum) {
```

Comment:
This is a misnomer. `and_append_overloaded` is in reference to `__torch_function__` accumulation. But Scalars can't be torch function'ed in this way. You don't even take `overloaded_args` as an argument.

Reply:
renamed